script.js

```diff
@@ -1,4 +1,111 @@
+var Dice1;
+var Dice2;
+var orderMode="";
+var playerMode=1;
+var player1Score=0;
+var player2Score=0;
+var outputmsg1= "Player 1, you chose Dice 1 first.<br>" + "Your number is ";
+var outputmsg2= "Player 1, you chose Dice 2 first.<br>" + "Your number is ";
+var outputmsg3= "Player 2, you chose Dice 1 first.<br>" + "Your number is ";
+var outputmsg4= "Player 2, you chose Dice 2 first.<br>" + "Your number is ";
+var outputmsg5= "<br> It is now Player 1's turn. <br>";
+var outputmsg6= "<br> It is now Player 2's turn. <br>";
+var outputmsg7="The current leaderboard is Player 1 wins with score ";
+var outputmsg8="The current leaderboard is Player 2 wins with score " ;
+var outputmsg9="The current leaderboard is both players tie with score ";
+var outputmsg10=".<br> The Player 1's score is ";
+var outputmsg11=".<br> The Player 2's score is "
+
 var main = function (input) {
```

Comment:
Would be good if you can add more comments to explain your line of thought

Reply:
Take turn for each player to roll two dice and then allow the player to decide which dice comes first to form a two-digit number. Accumulate every time each player throws a dice and compare the the results and show leaderboard.